Route/PathInfoBuilderSlashes.php

```diff
@@ -26,11 +26,12 @@ public function build(AdminInterface $admin, RouteCollection $collection)
     {
         $collection->add('list');
         $collection->add('create');
-        $collection->add('batch', null, array(), array('_method' => 'POST'));
+        $collection->add('batch', null, array(), array(), array(), '', array(), array('POST'));
```

Comment:
sonata admin 2 is not compatible with sf 3 AFAIK, which means you can leave the code as it was before for Sonata 2.x